src/nix/upgrade-nix.md

```diff
@@ -1,33 +1,12 @@
 R""(
 
-# Examples
-
-* Upgrade Nix to the stable version declared in Nixpkgs:
-
-  ```console
-  # nix upgrade-nix
-  ```
-
-* Upgrade Nix in a specific profile:
-
-  ```console
-  # nix upgrade-nix --profile ~alice/.local/state/nix/profiles/profile
-  ```
-
 # Description
 
-This command upgrades Nix to the stable version.
-
-By default, the latest stable version is defined by Nixpkgs, in
-[nix-fallback-paths.nix](https://github.com/NixOS/nixpkgs/raw/master/nixos/modules/installer/tools/nix-fallback-paths.nix)
-and updated manually. It may not always be the latest tagged release.
-
-By default, it locates the directory containing the `nix` binary in the `$PATH`
-environment variable. If that directory is a Nix profile, it will
-upgrade the `nix` package in that profile to the latest stable binary
-release.
+This command isn't available in Determinate Nix but is present in order to guide
+users to the new upgrade path.
 
-You cannot use this command to upgrade Nix in the system profile of a
-NixOS system (that is, if `nix` is found in `/run/current-system`).
+Use `sudo determinate-nixd upgrade` to upgrade Determinate Nix on systems that
+manage it imperatively. In practice, this is any system that isn't running
+NixOS.
```

Comment:
```suggestion
Use `sudo determinate-nixd upgrade` to upgrade Determinate Nix on systems that manage it imperatively.
In practice, this is any system that isn't running NixOS.
Run `sudo determinate-nixd upgrade` for system-specific instructions.
```

Reply:
I've adjusted the formatting, but I think the last addition is a bit too repetitive. How does this look if I just address the nit?